docs/user/commandline_old_commands.rst

```diff
@@ -0,0 +1,72 @@
+Old Command Line Commands
+=========================
+
+In GeoMesa 1.3.0 some command names have changed. Please reference the chart below for more information.
+
+Accumulo Command Line Tools
+---------------------------
+
++---------------------+--------------------+
+| Old                 | New                |
++=====================+====================+
+| add-attribute-index	| add-attributeindex |
++---------------------+--------------------+
+| add-index         	| add-index          |
++---------------------+--------------------+
+| create	            | create-schema      |
++---------------------+--------------------+
+| deletecatalog	      | delete-catalog     |
++---------------------+--------------------+
+| deletefeatures	    | delete-features    |
++---------------------+--------------------+
+| deleteraster	      | delete-raster      |
++---------------------+--------------------+
+| describe	          | get-schema         |
++---------------------+--------------------+
+| env 	              | env                |
++---------------------+--------------------+
+| explain	            | explain            |
++---------------------+--------------------+
+| export             	| export             |
++---------------------+--------------------+
+| genavroschema	      | gen-avroschema     |
++---------------------+--------------------+
+| getsft	            | get-sft            |
++---------------------+--------------------+
+| help               	| help               |
++---------------------+--------------------+
+| ingestraster	      | ingest-raster      |
++---------------------+--------------------+
+| keywords          	| keywords           |
++---------------------+--------------------+
+| list	              | get-names          |
++---------------------+--------------------+
+| queryrasterstats	  | query-rasterstats  |
++---------------------+--------------------+
+| removeschema	      | remove-schema      |
++---------------------+--------------------+
+| tableconf         	| config-table       |
++---------------------+--------------------+
+| version           	| version            |
++---------------------+--------------------+
+
+Kafka Command Line Tools
+------------------------
+
++---------------------+--------------------+
+| Old                 | New                |
++=====================+====================+
+| create	            | create-schema      |
```

Comment:
did you view the generated docs and make sure these tables display ok? rst is real finicky.